exporter/fileexporter/README.md

```diff
@@ -2,9 +2,8 @@
 
 This exporter will write the pipeline data to a JSON file.
 The data is written in Protobuf JSON encoding
-(https://developers.google.com/protocol-buffers/docs/proto3#json).
-Note that there are no compatibility guarantees for this format, since it
-just a dump of internal structures which can be changed over time.
+(https://developers.google.com/protocol-buffers/docs/proto3#json)
+using [OpenTelemetry protocol](https://github.com/open-telemetry/opentelemetry-proto).
```

Comment:
Keep the message that compatibility is not guaranteed. JSON will print the field names which we don't guarantee to be backwards compatible, but binary format is.